haystack/retriever/dense.py

```diff
@@ -292,4 +271,4 @@ def embed_passages(self, texts: List[str]) -> List[np.array]:
         :return: embeddings, one per input passage
         """
 
-        return self.embed(texts)
+        return self.embed(texts)
```

Comment:
I know the others do not always do so, too, but lets always add a newline at the end of file, so when we add code the last line doesnt show up as changed in git diff.